config/routes.rb

```diff
@@ -12,6 +12,8 @@
       resources :comments
     end
   end
+
+  devise_for :users, controllers: { sessions: 'users/sessions' }
   
   get '/posts/:id', to: 'posts#show'
```

Comment:
Please correct the route config to access the sign_in and sign_up page like asked in the requirement.

<img width="747" alt="image" src="https://user-images.githubusercontent.com/7243132/193415287-9109784e-ae3f-4409-bba1-2531e0c52b85.png">
<img width="747" alt="image" src="https://user-images.githubusercontent.com/7243132/193415292-4bcf945b-040b-443b-8496-1f317ce9df4f.png">